tools/reactive-controllers/src/FocusGroup.ts

```diff
@@ -163,6 +163,26 @@ export class FocusGroupController<T extends HTMLElement>
         }
     }
 
+    changeDefaultItemFocus(): void {
```

Comment:
I think it would make better sense for this to be a reaction to something we can observe rather than something we call imperatively. Can you think of paths to leverage a [Mutation Observer](https://developer.mozilla.org/en-US/docs/Web/API/MutationObserver) here? A great way to do so in this context is via [this Lit Labs package](https://github.com/lit/lit/blob/main/packages/labs/observers/src/mutation-controller.ts).

Reply:
Yes now we are observing this method instead of calling it imperatively.
```
new MutationController(this, {
            config: {
                childList: true,
                subtree: true,
            },
            callback: () => {
                this.rovingTabindexController.changeDefaultItemFocus();
            },
        });
```